src/NHSD.BuyingCatalogue.Organisations.Api/Repositories/OdsRepository.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Flurl;
+using Flurl.Http;
+using NHSD.BuyingCatalogue.Organisations.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Organisations.Api.Repositories
+{
+    internal sealed class OdsRepository : IOdsRepository
+    {
+        private readonly string _buyerOrganisationRoleId;
+        private readonly string _odsApiBaseUrl;
+
+        public OdsRepository(string odsApiBaseUrl)
+        {
+            _buyerOrganisationRoleId = "RO98";
```

Comment:
Did you consider moving the role ID(s) to config?

Reply:
👍 Now that the amount of Ids has grown, I'll be definitely moving them to config.